package.json

```diff
@@ -279,7 +278,7 @@
     "window-size": "^1.1.0"
   },
   "optionalDependencies": {
-    "net-keepalive": "^1.3.3",
+    "net-keepalive": "^1.3.6",
```

Comment:
1.3.6 is actually the version already in the shrinkwrap, this just updates package.json to reflect that